flake.nix

```diff
@@ -165,10 +165,6 @@
       system: let
         overlays = [
           poetry2nix.overlay
```

Comment:
I think we should remove the global usage of overlays and reintroduce it only where we need to (I guess in this case it is `jupyterlabEnvWrapped`).